specification/reservations/resource-manager/readme.md

```diff
@@ -35,7 +35,7 @@ These settings apply only when `--tag=package-2020-10-25` is specified on the co
 
 ``` yaml $(tag) == 'package-2020-10-25'
 input-file:
-  - Microsoft.Capacity/stable/2017-11-01/reservations.json
+  - Microsoft.Capacity/preview/2020-10-01-preview/reservations.json
```

Comment:
Changing this makes this tag `package-2020-10-25` a preview package, which results a error in go SDK package validation. In go SDK, a preview package has the different patterns of path from the stable packages.
Is this change really necessary? If not, could we revert this change? 
If this is necessary, I recommend that we make a new tag for the preview swagger.

Reply:
Yes, we need to include the right version of the Reservation SDK, as the previous release of SDK was - preview/2020-10-01-preview, so we cannot go back and it breaks all the tests built with the SDK.

If I create a tag ``` yaml $(tag) == 'package-2020-10-25-preview' for Go, as it's giving error and use package-2020-10-25 for others, will it work?

Reply:
Added following config to Readme.go.md 
### Tag: package-2020-10-25 and go

These settings apply only when `--tag=package-2020-10-25 --go` is specified on the command line.
Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.

``` yaml $(tag)=='package-2020-10-25' && $(go)
output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2020-10-25/$(namespace)
```
 I have added /preview path for package-2020-10-25 in readme.go.md .

Reply:
Thanks for the help. Go SDk is building fine. Please merge the changes.

Reply:
Well... this change resolves the CI failure in this PR, but it introduces a big breaking change to go SDK. 
By changing the package path in readme.go.md, we are renaming one package from stable to preview, a.k.a. removing one stable package and adding a new preview package. This would be considered as a breaking change.
Some customers might have concerns of using preview packages because breaking changes might be introduced to it in any time.
Is it possible that we keep this tag `package-2020-10-25` unchanged, and introduce a new tag with this new preview swagger? This would leave the old package unchanged.

Reply:
The previously released package was using the same preview version.
The last stable version was released in 2017. Customers have already moved to preview versions, so there shouldn't be any impact. 
The customers have been using the previous preview version of package, as the last stable version release was in 2017. Going back to 2017 version of reservation API,  breaks all the packages and the tests associated with it. And it doesn't help customers, as it will break their automation, as they have moved forward. That would be breaking change.

There is not much change from previous version of package and current package-
1. Version no updated and preview feature API is removed.

**The tag package-2020-10-25 is introduced for this GA release. It doesn't exists or released in any form. It's new package getting created.**

Please proceed forward with the merge, so that we can go GA by 1/19.
Thanks for help for time release of SDKs.